docs/configs/nav/nav_v0.2.yml

```diff
@@ -56,3 +56,6 @@ nav:
   - How to Deploy and Run with Helm?: onos-gui/docs/deployment.md
 - ONOS Library Packages:
     - Logging: onos-lib-go/docs/logging.md
+- TLA+ Conformance Monitoring Tools:
+    - How to Deploy and Run with Helm?: tlaplus-monitor/docs/README.md
```

Comment:
If we do the above, just name this the same as the title - `Conformance Monitoring with TLA+`. It introduces the concept and then talks about deployment